roles/openshift_sdn/defaults/main.yml

```diff
@@ -1,6 +1,13 @@
 ---
-openshift_node_image_dict:
-  origin: 'openshift/node'
-  openshift-enterprise: 'openshift3/node'
-oreg_host: "{{ oreg_url.split('/')[0] ~ '/' if (oreg_url is defined and '.' in oreg_url.split('/')[0]) else '' }}"
-osn_image: "{{ oreg_host }}{{ openshift_node_image_dict[openshift_deployment_type | default('origin')] }}:{{ openshift_image_tag | default('latest') }}"
+l_openshift_master_images_dict:
```

Comment:
This variable and others could use a better name since these don't have anything to do with master images.